app/containers/ActionSheet/ActionSheet.tsx

```diff
@@ -32,33 +26,32 @@ const ActionSheet = React.memo(
 		const bottomSheetRef = useRef<BottomSheet>(null);
 		const [data, setData] = useState<TActionSheetOptions>({} as TActionSheetOptions);
 		const [isVisible, setVisible] = useState(false);
-		const { height } = useDimensions();
-		const { isLandscape } = useOrientation();
-		const insets = useSafeAreaInsets();
-
-		const maxSnap = Math.min(
-			// Items height
-			ITEM_HEIGHT * (data?.options?.length || 0) +
-				// Handle height
-				HANDLE_HEIGHT +
-				// Custom header height
-				(data?.headerHeight || 0) +
-				// Insets bottom height (Notch devices)
-				insets.bottom +
-				// Cancel button height
-				(data?.hasCancel ? CANCEL_HEIGHT : 0),
-			height - MIN_SNAP_HEIGHT
+		const { width, height } = useWindowDimensions();
+		const isLandscape = width > height;
+		const animatedContentHeight = useSharedValue(0);
+		const animatedHandleHeight = useSharedValue(0);
+		const animatedSnapPoints = useDerivedValue(() => {
+			if (data?.snaps) {
+				return data.snaps;
+			}
+			const contentWithHandleHeight = animatedContentHeight.value + animatedHandleHeight.value;
+			if (contentWithHandleHeight === 0) {
+				return ['25%'];
```

Comment:
This is only to make the lib happy, as it requires a valid snap for the first render.
It's similar to `INITIAL_SNAP_POINTS` https://github.com/gorhom/react-native-bottom-sheet/blob/218e006a4a015625fcf424b99f432319daf88aad/src/hooks/useBottomSheetDynamicSnapPoints.ts#L22